src/store/actions.js

```diff
@@ -260,46 +260,59 @@ export default {
 			commit("setEmailPreviewInited", false);
 		}
 	},
-	async initHtmlImport() {
-		//  if (state.htmlImportInited === true || state.htmlImportInited === "pending") return;
-		//  await dispatch("waitForSdkToBeInited");
-		// 	commit("setHtmlImportInited", "pending");
-		// 	try {
-		// 		Vue.prototype.$chamaileon.htmlImport = await Vue.prototype.$chamaileon.createFullscreenPlugin({
-		// 			// ...this.$store.getters.getHtmlImportConfigObject,
-		// 			plugin: "import",
-		// 			hooks: {
-		// 				cancel: () => {
-		// 					console.log("TODO CANCEL");
-		// 					Vue.prototype.$chamaileon.htmlImport.hide();
-		// 				},
-		// 				close: () => {
-		// 					console.log("TODO CLOSE");
-		// 					Vue.prototype.$chamaileon.htmlImport.hide();
-		// 				},
-		// 				importReady: (message) => {
-		// 					console.log("TODO onButtonClicked");
-		// 					// eslint-disable-next-line no-unused-vars
-		// 					const template = {
-		// 						content: message.document,
-		// 					};
-		// 					console.log(message.document);
-		// 					Vue.prototype.$chamaileon.htmlImport.hide();
-		// 				},
-		// 				// eslint-disable-next-line no-unused-vars, require-await
-		// 				onButtonClicked: async ({ buttonId, data }) => {
-		// 					console.log("TODO onButtonClicked");
-		// 				},
-		// 			},
-		// 		});
-		// 		console.log(Vue.prototype.$chamaileon);
-		// 		commit("setHtmlImportInited", true);
-		// 	} catch (error) {
-		// 		console.error("Failed to initialize html import: ", error);
-		// 		Vue.prototype.$chamaileon.htmlImport = null;
-		// 		commit("setHtmlImportInited", false);
-		// 	}
-		// }
+	async initHtmlImport({ getters, commit, state, dispatch }) {
+		if (state.htmlImportInited === true || state.htmlImportInited === "pending") return;
+		commit("setHtmlImportInited", "pending");
+		await dispatch("waitForSdkToBeInited");
+		try {
+			Vue.prototype.$chamaileon.htmlImport = await Vue.prototype.$chamaileon.createFullscreenPlugin({
+				...getters.getHtmlImportConfigObject,
+				plugin: "import",
+				settings: {
+					replaceImages: true,
+					showReplaceSwitch: true,
+				},
+				hooks: {
+					cancel: () => {
+						Vue.prototype.$chamaileon.htmlImport.hide();
+					},
+					close: () => {
+						Vue.prototype.$chamaileon.htmlImport.hide();
+					},
+					onReplaceImage: src => src,
+					onImport: async ({ html }) => {
+						if (!html) throw new Error("HTML missing");
+						const response = await fetch("https://sdk-api.chamaileon.io/api/v1/emails/import", {
```

Comment:
We agreed on that this won't be here and it only will function as a mock import and not as a fully functioning import

Or if you want to be really fancy then we can mock this on the demo backend and always send back the same result json from there